sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -174,7 +174,13 @@ def teardown(
         tables: Sequence[FeatureView],
         entities: Sequence[Entity],
```

Comment:
Not a fan of using the variable name "tables" here instead of "feature_views" but I suppose that's a decision feast made when implementing the online store interface.

Reply:
yes, it can be a bit confusing